Large File Support (#1383)
appwrite/appwrite: docs/references/storage/create-file.md

> Create a new file. The user who creates the file will automatically be assigned to read and write access unless they have passed custom values for read and write arguments.

Reviewer: ```suggestion
Create a new file. The user that creates the file will automatically be granted read and write permissions unless they have passed custom values for read and write permissions.
```

> Larger files should be uploaded using multiple requests with the [content-range](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Range) header to send a partial request with a maximum supported chunk of `2MB`. The `content-range` header values should always be in bytes.

Reviewer 2: Was it 5 or 2 MB?

Author: Ongoing, as S3 only supports at least 5 MB chunks, we might rethink this. Right now the payload size is 5MB, but chunk has to be smaller to accomodate other data in the request

> When the first request is sent, the server will return the **File** object, and the subsequent part request must include the file's **id** in `x-appwrite-upload-id` header to allow the server to know the partial upload is for the existing file and not for a new one.

Reviewer: ```suggestion
When the first request is sent, the server will return the **File** object, and the subsequent part request must include the file's **id** in `x-appwrite-upload-id` header to allow the server to know that the partial upload is for the existing file and not for a new one.
```

> If you're creating a new file using one of the Appwrite SDKs, the entire chunking logic will be managed by the SDK internally.

Reviewer: ```suggestion
If you're creating a new file using one the Appwrite SDKs, all the chunking logic will be managed by the SDK internally.
```
